elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/integration/AggregationDataStoreIntegrationTest.java

```diff
@@ -913,12 +913,20 @@ public void incompleteClientFilterTest() {
         String expectedError = "Querying deliveryDetails requires a mandatory filter:"
                 + " month&gt;={{start}};month&lt;{{end}}";
         when()
-        .get("/deliveryDetails?month&gt;=2020-08")
```

Comment:
Why did we change this?  Wouldn't this have still failed?

Reply:
syntax was wrong, so corrected it. Without the change error returned was due to queryParams validation.